recipe/meta.yaml

```diff
@@ -81,5 +81,6 @@ about:
 
 extra:
   recipe-maintainers:
+    - mtjrider
```

Comment:
@mtjrider please confirm you accept this change

Reply:
(this is standard process of adding new maintainers btw)

Reply:
Yes